internal/pkg/remote/services.go

```diff
@@ -22,20 +27,19 @@ func (a *StorageApi) GetServices() ([]interface{}, error) {
 	return nil, response.Err()
 }
 
-func (a *StorageApi) getServiceApiUrl(serviceName string) (string, error) {
-	services, err := a.GetServices()
+func (a *StorageApi) ServicesUrlById() (map[ServiceId]ServiceUrl, error) {
```

Comment:
Odstranene metody `GetEncryptionApiUrl`, `GetSchedulerApiUrl`.
Pridana metoda `ServicesUrlById`.

Ide o to, aby sa API requests na ziskanie vsetkych `services` volal iba raz.
Vo vyslednom kode (po dalsich PRs) to bude lazy.

Ak nejaka operacia bude potrebovat napr. `encryption` API, tak sa posle requests na ziskanie `services`, ... 
Ak nejaka dalsia operacia bude potrebovat `scheduler` API, tak sa request uz neposle, lebo `services` su uz nacitane.
Toto je len priprava (tj. samo o sebe to nic nevylepsuje).